test/to_vec.jl

```diff
@@ -67,8 +66,8 @@ end
         test_to_vec(UpperTriangular(randn(T, 13, 13)))
         test_to_vec(Diagonal(randn(T, 7)))
         test_to_vec(DummyType(randn(T, 2, 9)))
-        test_to_vec(SVector{2, T}(1.0, 2.0))
-        test_to_vec(SMatrix{2, 2, T}(1.0, 2.0, 3.0, 4.0))
+        test_to_vec(SVector{2, T}(1.0, 2.0); check_inferred = false)
+        test_to_vec(SMatrix{2, 2, T}(1.0, 2.0, 3.0, 4.0); check_inferred = false)
```

Comment:
```suggestion
        test_to_vec(SVector{2, T}(1.0, 2.0); check_inferred=false)
        test_to_vec(SMatrix{2, 2, T}(1.0, 2.0, 3.0, 4.0); check_inferred=false)
```
Bluetyle spacing in kwargs

Reply:
changed in other places as well now